roles/common/templates/etc/sshd_config

```diff
@@ -0,0 +1,92 @@
+# Package generated configuration file
```

Comment:
Can you give us some details of what all you're changing here vs what ships with Ubuntu (and we tweak with lineinfile)?